README.md

```diff
@@ -36,9 +34,11 @@ to set up this repo properly.**
     -   Ask one of the repo code owners to approve the PR
     -   Upon approval, another GitHub workflow will run to grant you the IAM
         permissions or run the on-demand `gcloud` commands
+    -   The result will be posted as PR comments
 
-4.  Close the PR after you no longer need the access. The PR will automatically
-    be closed after X hours (TODO #8).
+4.  The AOD request PR cannot and should not be merged. Close the PR after you
+    no longer need the access. The PR will automatically be closed after X hours
+    (TODO #8).
```

Comment:
we can remove todo 8, it is now added - https://github.com/abcxyz/aod-template/blob/main/.github/workflows/expire.yml, currently cleanup will be triggered daily at 1am, An AOD PR last updated 24 hours ago will be closed.